src/app/embed/adapters/merge.adapter.ts

```diff
@@ -0,0 +1,36 @@
+import {Injectable} from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+import { ReplaySubject } from 'rxjs';
+import { QSDAdapter } from './qsd.adapter'
+import { FeedAdapter } from './feed.adapter'
+import { AdapterProperties, hasMinimumParams, DataAdapter, getMergedValues } from './adapter.properties'
+
+const NO_EMIT_YET = Symbol();
+
+@Injectable()
+export class MergeAdapter {
+  private adapters: DataAdapter[]
+  constructor(
+    private FeedAdapter: FeedAdapter,
+    private QSDAdapter:  QSDAdapter
+	) { 
+    this.adapters = [this.QSDAdapter, this.FeedAdapter]
```

Comment:
Actually, as I re-read this, it seems the `EMBED_FEED_URL_PARAM` "uf" isn't actually part of the QSAdapter.  (I was thinking it was also used as the subscription url - but it's not).  So this should be fine.  I won't get an extra request to the XML feed unless I set a `uf`.